drivers/display/display_ili9163.h

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2022 Tavish Naruka <tavishnaruka@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_DRIVERS_DISPLAY_DISPLAY_ILI9163_H_
+#define ZEPHYR_DRIVERS_DISPLAY_DISPLAY_ILI9163_H_
+
+#include <device.h>
+
+/* Commands/registers. */
+#define ILI9163_GAMSET 0x26
+#define ILI9163_FRMCTR1 0xB1
+#define ILI9163_PGAMCTRL 0xE0
+#define ILI9163_NGAMCTRL 0xE1
+#define ILI9163_PWCTRL1 0xC0
+#define ILI9163_PWCTRL2 0xC1
+#define ILI9163_VMCTRL1 0xC5
+#define ILI9163_VMCTRL2 0xC7
+#define ILI9163_GAMADJ 0xF2
+#define ILI9163_DISPINV 0xB4
```

Comment:
some of these register definitions are not needed, see https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/display/display_ili9xxx.h. Only those used in `display_ili9163.c` should be defined here.

Reply:
Since this file is explicitly sending the command for ILI9163_GAMSET, I added it here just like it is in `display_ili9340.h` and `display_ili9341.h`.